### PR TITLE
Bind grn_obj_is_accessor()

### PR DESCRIPTION
**Pull request by myokoym**
See #110

@kou Could you review it?

### CODE REVIEW

File: ext/groonga/rb-grn-object.c [hunk 1]
```diff
@@ -1656,6 +1656,32 @@ rb_grn_object_scorer_procedure_p (VALUE self)
...
 }
 
+/*
+ * Checks whether the object is accessor or not.
+ *
+ * @overload accessor?
+ *   @return [Boolean] `true` if the object is accessor,
+ *     `false` otherwise.
+ *
+ * @since 5.1.0
```
Comment (kou) [inline ext/groonga/rb-grn-object.c]: The next version is 5.1.1. We use the same version number as Groonga.

Comment (myokoym) [inline ext/groonga/rb-grn-object.c]: I've fixed it.

**Comment by myokoym** [comment]
I'll squash the commits and merge the pull request.

**Comment by myokoym** [comment]
Merged. Next, I'll send a pull request for `grn_obj_is_key_accessor()`.

**Pull request merged**